content/setup/local-with-vagrant/_index.md

```diff
@@ -0,0 +1,198 @@
++++
+title = "Local with Vagrant"
+date = 2020-06-08T16:16:15+05:30
+draft = false
+weight = 20
+toc = true
++++
+
+An easy way to try out Tinkerbell locally is via [Vagrant](https://www.vagrantup.com).
+
+Vagrant wraps and glues together different technologies to help you write as
+code repetitive environment mixing virtualization or cloud providers such as
+VirtualBox, libvirt, AWS, GCP with provisioning techniques like Puppet, Ansible,
+Bash.
+
+In this context we use VirtualBox or libvirt as a virtualization platform and
+bash for provisioning all the required tools and configurations.
+
+Following this tutorial, you will end up with a provisioner up and running and a
+worker ready to handle actions and workflows.
+
+## Prerequisites
+
+- [The host's processor should support virtualization](https://www.cyberciti.biz/faq/linux-xen-vmware-kvm-intel-vt-amd-v-support/)
+- [Vagrant](https://www.vagrantup.com/downloads) is installed
+- Either [VirtualBox](https://www.virtualbox.org/) or [libvirtd](https://libvirt.org/) is installed and configured.
```

Comment:
Isn't libvirtd for Linux users? Can we make it clearer?

Reply:
In theory you can run both in both platforms. But I never tried libvirt https://libvirt.org/platforms.html